lib/get-client.js

```diff
@@ -95,19 +95,14 @@ const handler = (retry, limit, globalThrottler, endpoint) => ({
 
 module.exports = ({
   githubToken,
-  githubUrl,
-  githubApiPathPrefix,
+  githubUrl = '',
+  githubApiPathPrefix = '',
```

Comment:
`githubApiPathPrefix` is already defaulted to `''` in [resolve-config.js#L15](https://github.com/semantic-release/github/blob/master/lib/resolve-config.js#L15) so it shouldn't be required to default it here again.